src/main/java/org/espeakng/jeditor/gui/EventHandlers.java

```diff
@@ -374,13 +375,13 @@ public void actionPerformed(ActionEvent arg0) {
 	ActionListener compileDictionary = new ActionListener() {
 		public void actionPerformed(ActionEvent e) {
 			if (e.getSource() == mainW.mntmCompileDictionary) {
+				fileChooser.setCurrentDirectory(new File(dataPath + "/dictsource/"));
 				if (fileChooser.showOpenDialog(mainW) == JFileChooser.APPROVE_OPTION) {
-					Runtime rt = Runtime.getRuntime();
-					try {
-						rt.exec("javac " + fileChooser.getSelectedFile().getParent());
-					} catch (Exception exception) {
-						exception.printStackTrace();
-					}
+					String cmd = "export ESPEAK_DATA_PATH="+ dataPath +
+							"; cd " + fileChooser.getSelectedFile().getParent() +
+							" && " + dataPath + "/src/espeak-ng --compile=" + 
+							fileChooser.getSelectedFile().getName().split("_")[0];
+					org.espeakng.jeditor.utils.CommandUtilities.main(null, cmd);
```

Comment:
Don't use full path for callling method from neighbour package.
Merge master branch into the feature branch (e.g. https://stackoverflow.com/questions/16955980/git-merge-master-into-feature-branch) and rewrite your code using new CommandUtilities API
e.g.
`CommandUtilities.executeCmd(cmd)`